matching-notification-ui/src/main/resources/MatchingNotification/MatchesTable.xml

```diff
@@ -201,22 +207,22 @@ $xwiki.jsx.use('PhenoTips.Widgets')##
-                  &lt;option value="has_exome"&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.filterHasExome'))&lt;/option&gt;
-                &lt;/select&gt;
+              &lt;li&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.notifiedStatusFilter'))&lt;span class="fa fa-info-circle xHelpButton" title="$escapetool.xml($services.localization.render('phenotips.myMatches.notifiedFilter.hint'))"&gt;&lt;/span&gt;&lt;br&gt;
+                &lt;label&gt;
+                  &lt;input name="notified-filter" type="checkbox" value="notified" #if(!$hasAdmin) checked #end&gt; $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.yes'))&lt;br&gt;
+                &lt;/label&gt;&lt;label&gt;
+                  &lt;input name="notified-filter" type="checkbox" value="unnotified" checked &gt; $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.no'))&lt;/label&gt;&lt;br&gt;
+              &lt;/li&gt;
+              &lt;li&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.contactedStatusFilter'))&lt;span class="fa fa-info-circle xHelpButton" title="$escapetool.xml($services.localization.render('phenotips.myMatches.contactedFilter.hint'))"&gt;&lt;/span&gt;&lt;br&gt;
+                &lt;label&gt;
+                  &lt;input name="contacted-filter" type="checkbox" value="contacted" #if(!$hasAdmin) checked #end&gt; $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.yes'))&lt;br&gt;
+                &lt;/label&gt;&lt;label&gt;
+                  &lt;input name="contacted-filter" type="checkbox" value="uncontacted" checked &gt; $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.no'))&lt;/label&gt;&lt;br&gt;
               &lt;/li&gt;
               &lt;li class="hidden"&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.accessFilter'))&lt;br&gt;
                 &lt;label&gt;&lt;input type="checkbox" name="access-filter" value="owner" checked /&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.owner'))&lt;/label&gt;&lt;br&gt;
                 &lt;label&gt;&lt;input type="checkbox" name="access-filter" value="view" checked /&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.view'))&lt;/label&gt;
               &lt;/li&gt;
-              &lt;li&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.solvedFilter'))&lt;br&gt;
-                &lt;label&gt;&lt;input type="checkbox" name="solved-filter" value="hide"/&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.solved'))&lt;/label&gt;
-              &lt;/li&gt;
             &lt;/ul&gt;
           &lt;/div&gt;
         &lt;/div&gt;
```

Comment:
There is still a problem mentioned in the original issue that terms with the same name but different status are still highlighted as matching. E.g. `Visual impairment` and `NO Visual impairment`. Easy to see on any existing match by adding any phenotype to one patient and marking that phenotype as NO for the other